cypress/e2e/home-spec.cy.js

```diff
@@ -0,0 +1,46 @@
+describe('rancid tomatillos homepage user flows', () => {
+
+  beforeEach(() => {
+    cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies', {
+      statusCode: 200,
+      fixture: '.././fixtures/all-movies-test.json'
+    })
+    .visit('http://localhost:3000');
+  });
+  
+  it('should display all navbar elements upon navigating to the app', () => {
+    cy.contains('h1', 'RANCID')
+    cy.contains('h2', 'TOMATILLOS')
+    cy.get('img[name=tomato-image]')
+      .should('be.visible')
+    cy.contains('div', 'Welcome, user!')
+    cy.location('pathname').should('eq', '/');
+  });
+  
+  it('should display filter ribbon', () => {
+    cy.get('input[name=searchbar]')
+    cy.get('img[name=cross-icon]')
+      .should('be.visible')
+  });
+  
+  it('should display all movie posters', () => {
+    cy.get('.all-movies-container').children()
+      .should('have.length', 4)
+    cy.get('img[name=movie-poster]')
+      .get('#436270')
+    cy.contains('h3', 'Black Adam (2022)')
+  });
+  
+  it('should direct a click to the clicked movie\'s details', () => {
+    cy.get('img[name=movie-poster]')
+      .get('#436270').click()
+      .url().should('include', '/436270')
```

Comment:
Hey, I feel a more appropriate test for urls should be an 'eq' instead of 'include'.
it'd go like 
.url().should('eq', [exact url]) 
given that we're using routes, this could be vital.

Reply:
I'm down for that! Would that test break when deployed though, since the url changes from `localhost`?

Reply:
I decided to change to the exact url. I think you're right 😺